src/alpine/3.19/amd64/Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM amd64/alpine:3.19
 
 # Install .NET and test dependencies
 RUN apk add --upgrade --no-cache \
+        apt \
```

Comment:
I don't know what this package is but it doesn't give you an `apt` command. The package manager for Alpine is `apk`. The MicroBuild script should support that if it doesn't already.

Reply:
Ah okay yeah the MicroBuild install script fails because it tries to install nuget via `sudo apt install nuget`. I'll see if I can file an issue against them for this.